<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>saveRecord</class>
 <widget class="QWidget" name="saveRecord">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>711</width>
    <height>475</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="pushButton_home">
       <property name="minimumSize">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">border-image: url(:/image/home.png);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_path">
       <property name="text">
        <string>&gt; 历史检测记录</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_exportDevImei">
       <property name="text">
        <string>导出合格设备IMEI</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_exportList">
       <property name="text">
        <string>导出列表</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_batch">
       <property name="text">
        <string>批次号</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox_batch"/>
     </item>
     <item>
      <widget class="QLabel" name="label_time">
       <property name="text">
        <string>检测时间</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDateEdit" name="dateEdit_startTime">
       <property name="calendarPopup">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>-</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDateEdit" name="dateEdit_endTime">
       <property name="calendarPopup">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_resultSelect">
       <property name="text">
        <string>检测结果</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox_resultSelect">
       <item>
        <property name="text">
         <string>全部</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>通过</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>不通过</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_find">
       <property name="text">
        <string>查询</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="0">
    <widget class="QTableWidget" name="tableWidget">
     <column>
      <property name="text">
       <string>模组型号</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>批次号</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>设备IMEI</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>设备SN</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>测试人</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>检测时间</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>模组版本</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>指示灯</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>蜂鸣器</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>按键</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>干簧管</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>联网</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>CSQ</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>检测结果</string>
      </property>
     </column>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
